gcp/api/server.py

```diff
@@ -1395,6 +1397,13 @@ def main():
     setup_gcp_logging('api-backend')
     logging.getLogger().addFilter(trace_filter)
 
+    # Profiler initialization. It starts a daemon thread which continuously
+    # collects and uploads profiles. Best done as early as possible.
+    try:
+      googlecloudprofiler.start(service="osv-api-profiler")
```

Comment:
Do you want to enable this only on the test instance first and then enable it on prod after verification? `if get_gcp_project() == _TEST_INSTANCE:`

Reply:
ah! thanks!